src/css/layout/meet_our_teachers.css

```diff
@@ -0,0 +1,50 @@
+.teacher-name {
+  margin-bottom: 8px;
+  color: var(--black);
+  text-align: center;
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 1.2; /* 120% */
+  letter-spacing: -0.02em;
+}
+
+.teacher-card{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.teacher-picture{
+    margin-bottom: 16px;    
+}
+
+.teacher-description{
+    text-align: center;
+    line-height: 1.71;
+}
+
+.teachers-list{
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+}
+
+@media screen and (min-width: 768px){
+    .teachers-list{
+        flex-direction: row;
+        flex-wrap: wrap;
+        gap: 32px;
+    }
+    .teacher-card{
+        width: calc((100% - 32px) / 2);
+    }   
+}
+
+@media screen and (min-width: 1280px){
+    .teachers-list{
+         flex-wrap: nowrap;        
+    }
+    .teacher-card{
+        width: calc((100% - 32px) / 4);
+    }
+}
```

Comment:
It seems like not all styles has been added. Please add styles for the title and section text